activities/FoodChain.activity/activity/activity.info

```diff
@@ -1,6 +1,15 @@
 [Activity]
 name = FoodChain
 activity_version = 4
+license = ASL 2.0
 bundle_id = org.olpcfrance.FoodChain
 exec = sugar-activity-web
 icon = activity-icon
+summary = FoodChain is a pedagogical game to learn name of animals.
+tags = Science;Game
+repository = https://github.com/llaske/FoodChain
+url = http://activities.sugarlabs.org/en-US/sugar/addon/4612
+screenshots = http://activities.sugarlabs.org/en-US/sugar/images/t/902/1352414186 http://activities.sugarlabs.org/en-US/sugar/images/t/904/1352414187
```

Comment:
As @quozl rightfully mentioned, did you try running the activity in Sugar before adding screenshots? 
Also, the screenshots that you added are different from the screenshots present on the url's that you listed.